.github/workflows/desktop-deploy.yaml

```
@@ -1,113 +1,117 @@
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - macos-15-intel # x86 (Intel) macOS
+          - ubuntu-latest
+          - ubuntu-24.04-arm
+          - windows-latest # (don't release, just build artifacts)
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v1
```

actionlint (1.7.7)

the runner of "actions/checkout@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)